src/test/java/millom/sandbox/Utility/WeatherDataUtility.java

```diff
@@ -0,0 +1,48 @@
+package millom.sandbox.Utility;
+import java.util.Arrays;
+import java.util.List;
+import millom.sandbox.pojo.Descriptions;
+import millom.sandbox.pojo.Sol;
+import millom.sandbox.pojo.NasaWeather;
+public class WeatherDataUtility {
+  private static final Descriptions DESCRIPTIONS = new Descriptions.DescriptionsBuilder("\\t    \\n\\t\\t\\tThe information contained into this file is provided by Centro de Astrobiologia\\n\\t\\t\\t(CAB) and is intended for outreach purposes only. Any other use is discouraged.\\n\\t\\t\\tCAB will take no responsibility for any result or publication made base on the \\n\\t\\t\\tcontent of this file. To access REMS scientific data, visit PDS at http://pds.nasa.gov.\\n\\n\\t\\t\\tThe environmental magnitudes given into this file are obtained from the values\\n\\t\\t\\tread by the Rover Environmental Monitoring Station (REMS) on board the Mars Science\\n\\t\\t\\tLaboratory (MSL) rover on Mars. This file provides the environmental magnitudes at REMS\\n\\t\\t\\tlocation, so MSL rover influences those magnitudes (rover position,\\n\\t\\t\\trover temperature, rover orientation, rover shade, dust depositions on the rover, etc.)\\n\\n\\t\\t\\tREMS does not take measurements continuously and it takes measurements at different\\n\\t\\t\\ttimes from one day to another. This fact has influence on the variation of the values\\n\\t\\t\\tgiven in this file from one day to another .\\n\\n\\t\\t\\tFor different reasons (instrument maintenance, instrument calibration, instrument degradation, etc.), \\n\\t\\t\\tsome or all of the magnitudes in this file may not be available.\\n\\t\\t",
```

Comment:
Such constructs are quite hard to maintain. With a proper builder it would be easier to both read and maintain.